<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKSP00075683" name="Test" tests="2" failures="2" timestamp="7 Feb 2017 13:45:30 GMT" time="18.484" errors="0">
  <testcase name="testEasy" time="0.886" classname="MAX.SeleniumHQ">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at MAX.SeleniumHQ.testEasy(SeleniumHQ.java:23)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testEasy -->
  <testcase name="valid_UserCredential" time="9.95" classname="MAX.SeleniumHQJUNIT">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at MAX.SeleniumHQJUNIT.valid_UserCredential(SeleniumHQJUNIT.java:37)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- valid_UserCredential -->
</testsuite> <!-- Test -->
